File: tests/java/src/main/java/com/escape/tests/cases/Case065EnrichTicket.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 065: enrich ticket payload for downstream business processing. */
public class Case065EnrichTicket {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        // Task: enrich ticket records and prepare transport-ready payload.
        String taskName = "enrich_ticket";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "ticket");
        payload.put("stage", "enrich");
        payload.put("input", raw);
        // SAFE: payload remains local; only primitive summary string is returned.
        return payload.get("task") + ":" + payload.get("input");
    }
}

File: tests/go/cases/case_065_enrich_ticket.go
package escape_tests

var retainedCase065 = []map[string]string{}

func Case065EnrichTicket(input string) string {
    // Task case 065: enrich ticket payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "enrich_ticket",
        "entity": "ticket",
        "stage": "enrich",
        "input": raw,
    }
    // SAFE: payload remains local; only primitive summary string is returned.
    return payload["task"] + ":" + payload["input"]
}
